### PR TITLE
Feat/remove ff from logs

### PR DESCRIPTION
**Pull request by ardourApeX**
Date: 29 Jan 2023

Developer Name: Joy Gupta

---

## Issue Ticket Number
- https://github.com/Real-Dev-Squad/website-dashboard/issues/682

## Description
Removed the feature flag which was earlier required to view all the extension request logs.

### Under Feature Flag

- [ ] Yes
- [x] No

### Database Changes

- [ ] Yes
- [x] No

### Breaking Changes

- [ ] Yes
- [x] No

### Development Tested?

- [x] Yes
- [ ] No

## Screenshots
<details>
<summary>Now audit logs are visible even without adding a feature flag.</summary>

https://github.com/Real-Dev-Squad/website-dashboard/assets/56365512/2c0ac930-6ebf-4eb2-a3b7-51d5c7398d46

</details>

<details>
<summary>Rest of the functionality is working fine after feature flag removal</summary>

https://github.com/Real-Dev-Squad/website-dashboard/assets/56365512/101d1b1e-5767-44db-94c3-2486c0d3a347

</details>

### Stats

Screenshot 2024-01-29 at 11 37 08 PM

### Coverage

Screenshot 2024-01-29 at 11 37 33 PM

### CODE REVIEW

File: extension-requests/script.js [hunk 1]
```diff
@@ -1039,33 +1033,31 @@ async function createExtensionCard(data) {
     logContainer.appendChild(creationLog);
   };
   // Adding log feature under dev flag
```
Comment (sahsisunny) [inline extension-requests/script.js:1035]: This comments  should  be removed.

File: extension-requests/script.js
Comment (sahsisunny) [inline extension-requests/script.js:1]: Can you remove comments from your changes?

File: __tests__/extension-requests/extension-requests.test.js [hunk 2]
```diff
@@ -818,7 +818,7 @@ describe('Tests the Extension Requests Screen', () => {
   });
 
   test('Checks the Request Number and request value element on Extension requests listing page', async () => {
-    const url = 'http://localhost:8000/extension-requests/?dev=true'; // Include the dev parameter in the URL
+    const url = 'http://localhost:8000/extension-requests'; // Include the dev parameter in the URL
```
Comment (ajoykumardas12) [inline __tests__/extension-requests/extension-requests.test.js:821]: Please remove these comments too.

Comment (ardourApeX) [reply __tests__/extension-requests/extension-requests.test.js:821]: fixed

File: extension-requests/script.js
Comment (ardourApeX) [reply extension-requests/script.js:1]: fixed

File: extension-requests/script.js (hunk 1 above)
Comment (ardourApeX) [reply extension-requests/script.js:1035]: fixed

**Review by ajoykumardas12 — APPROVED** [review]

**Pull request merged**